{{cookiecutter.project_slug}}/tests/CI_docker/context/{{cookiecutter.project_slug}}_ci.docker

```diff
@@ -17,13 +16,13 @@ RUN rm Mambaforge-$(uname)-$(uname -m).sh
 ENV PATH=/root/mambaforge/bin:${PATH}
 
 # Python packages from conda
-RUN mamba install -y python=3.9
+RUN mamba install -y python=3.12
 RUN mamba install -y pip
 RUN mamba init bash
 
 # copy some needed stuff to /root
 COPY *.yml /root/
-RUN mamba env create -n ci_env -f /root/environment_{{ cookiecutter.project_slug }}.yml python=3.10
+RUN mamba env create -n ci_env -f /root/environment_{{ cookiecutter.project_slug }}.yml python=3.12
```

Comment:
I think the option python=3.12 does not work anymore. It was removed for the latest version of mamba.

Reply:
with the changes from @jo-kno this got updated now